src/nvmesh-loaders.js

```diff
@@ -786,11 +794,7 @@ export class NVMeshLoaders {
       pos++ // skip null termination
       return new TextDecoder().decode(buffer.slice(startPos, pos - 1))
     } // readStr: read variable length string
-    const scalars = new Float32Array(nvert * nMaps)
-    const maps = []
-    // read Name of SRF
-    let pos = 9
-    // let filenameSRF = readStr();
+
```

Comment:
@jens-ox lets add an underscore as you [suggest here](https://github.com/niivue/niivue/issues/768)
```
let _filenameSRF = readStr();
```